pandas/compat/numpy/function.py

```diff
@@ -395,3 +396,21 @@ def validate_minmax_axis(axis: AxisInt | None, ndim: int = 1) -> None:
         return
     if axis >= ndim or (axis < 0 and ndim + axis < 0):
         raise ValueError(f"`axis` must be fewer than the number of dimensions ({ndim})")
+
+
+_validation_funcs = {
+    "median": validate_median,
+    "mean": validate_mean,
+    "min": validate_min,
+    "max": validate_max,
```

Comment:
Note that in main, the validate_mean, validate_min and validate_max were not used.

Reply:
if they're not used, let's delete them